pdf/delivery-slip.product-tab.tpl

```diff
@@ -89,7 +89,7 @@
 								</table>
 							{/if}
 
-							{if isset($customization.datas[Product::CUSTOMIZE_FILE]) && count($customization.datas[Product::CUSTOMIZE_FILE]) > 0}
+							{if !empty($customization.datas[Product::CUSTOMIZE_FILE])}
```

Comment:
I would add an `is_array` for checking that it is iterable.

Reply:
If defined, it's iterable. We are already in 2 foreach and they will crash before this line if we `$order_detail.customizedDatas` is null

Reply:
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Product.php#L5186